static/app/components/replays/breadcrumbs/replayTimelineSpans.tsx

```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 import styled from '@emotion/styled';
 
-import type {RawSpanType} from 'sentry/components/events/interfaces/spans/types';
 import {divide, flattenSpans} from 'sentry/components/replays/utils';
 import Tooltip from 'sentry/components/tooltip';
 import {tn} from 'sentry/locale';
 import space from 'sentry/styles/space';
+import type {ReplaySpan} from 'sentry/views/replays/types';
```

Comment:
This is now something the replay SDK produces and is span-like in properties, but not 100% compatible.